docs/DeveloperGuide.md

```diff
@@ -274,27 +274,56 @@ _{Explain here how the data archiving feature will be implemented}_
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
+| Priority | As a …​                                                                                                      | I want to …​                                                                                                                | So that I can…​                                                                     |
+|----------|--------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+| `* * *`  | Y1 CS Student                                                                                                | add a contact                                                                                                               | start tracking my contacts on this app                                              |
+| `* * *`  | Y1 CS student who prefers a clean address book with useful contacts only                                     | delete contacts                                                                                                             | remove contacts that I no longer need to keep track of                              |
+| `* * *`  | Y1 CS student who hates using a mouse or touchpad                                                            | access to all functionalities of the app with keyboard                                                                      | manage the contacts in the most efficient way                                       |
+| `* * *`  | Y1 CS student who wants to seek clarifications/actions from a professor regarding certain issues             | find contact details of the professor(s) for a course                                                                       | seek help/assistance with ease                                                      |
+| `* * *`  | Y1 CS student who is sick before a tutorial/lab/exam                                                         | find contact details of my TA                                                                                               | easily inform them about my absence on time                                         |
+| `* * *`  | Y1 CS student who is involved in any accident or emergency                                                   | find contact details of campus security                                                                                     | protect me and my friends' safety immediately                                       |
```

Comment:
maybe finding public contact can be ** since we are focusing on maunally added contact for mvp

Reply:
I have changed them to ** now, can you help me confirm if the changes are all correct and help spot any other issues in the current PR, thanks!